docs/toc.yml

```diff
@@ -1289,30 +1289,21 @@ items:
     - name: Use search options to find text in your Word add-in
       href: word/search-option-guidance.md
       displayName: Word
-
-- name: Patterns and practices
+- name: Samples
   items:
-  - name: Overview
-    href: overview/patterns-and-practices-overview.md
+  - name: Samples overview
```

Comment:
Should this be called "overview"? It's really the full sample list, with Open in Excel being an exception.